torch/csrc/Module.cpp

```diff
@@ -1506,6 +1510,14 @@ Call this whenever a new thread is created in order to propagate values from
   ASSERT_TRUE(set_module_attr("has_cuda", has_cuda));
   ASSERT_TRUE(set_module_attr("has_mps", has_mps));
   py_module.def("_is_mps_available", []() { return at::hasMPS(); });
+  py_module.def("_is_mps_on_macos_13_or_newer", []() {
+#ifdef USE_MPS
+    return at::mps::is_macos_13_or_newer();
+#else
+    return false;
+#endif
+  });
+  py_module.def("_is_mps_on_macos_13_or_newer", []() { return false; });
```

Comment:
This is from a previous iteration where the ifdef was somewhere else?

Reply:
Oops, artifact of refactoring. Fixed